webportal/src/app/cluster-view/hardware/hardware.component.js

```diff
@@ -250,7 +252,7 @@ const loadDiskUtilData = (prometheusUri, currentEpochTimeInSeconds, instanceList
 //
 
 const loadEthUtilData = (prometheusUri, currentEpochTimeInSeconds, instanceList, table) => {
-  const metricGranularity = '1m';
+  const metricGranularity = '5m';
```

Comment:
>5m [](start = 29, length = 2)

Suggest define this kind of numbers into a configuration class which contains all tunable parameters.

Reply:
Agree. This number should be consistent with that in Grafana. @YanjieGao : Do you know where I can find the configuration of this value in Grafana? I'll make the code change then.